spec/lib/notifications/extended_use_spec.rb

```diff
@@ -56,9 +56,11 @@
     end
 
     context 'when the notif quota has been exceeded' do
-      before { $redis.setex(key, described_class::QUOTA.to_s.to_i, nil) }
-
       it 'does not send a notif' do
+        expect(SMS).to receive(:send)
```

Comment:
I would move these into a `before` block as they are now setup code and replace `expect` with `allow`